### PR TITLE
Static typing fix

### PR DESCRIPTION
**Pull request by ellesmith88**
Added some missing types to the static type support.
Updated history to include this for next release.

### CODE REVIEW

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
@ellesmith88 I find it hard to guess what the allowed input parameters are just reading the static type description. Maybe the examples can be extended showing the different possibilities.

**Comment by ellesmith88** [comment]
@cehbrecht How are these updated examples?

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
> @cehbrecht How are these updated examples?

looks good :)

File: clisops/ops/subset.py [hunk 1]
```diff
@@ -50,47 +48,65 @@ def _subset(ds, args):
 
 
 def subset(
-    ds: Union[xr.Dataset, str, Path],
+    ds: Union[xr.Dataset, str],
```
Comment (Zeitsperre) [inline clisops/ops/subset.py:51]: Just want to make a note here that removing `Path` objects here will raise IDE warnings for users using Pathlib. The handling for them is covered on the new line 113.

Comment (ellesmith88) [reply clisops/ops/subset.py:51]: Thanks for pointing this out. I had thought that `Path` objects would raise an error from `utils.map_params` but as you said this is resolved on line 113, before `utils.map_params` is called. I'll put this back in

**Pull request merged**